tests/transactions/builder/test_transfer.py

```diff
@@ -27,6 +27,29 @@ def test_transfer_transaction():
     transaction.schnorr_verify()  # if no exception is raised, it means the transaction is valid
 
 
+def test_transfer_secondsig_transaction():
+    """Test if a transfer transaction with second signature gets built
+    """
+    transaction = Transfer(
+        recipientId='AGeYmgbg2LgGxRW2vNNJvQ88PknEJsYizC',
+        amount=200000000,
+    )
+    transaction.set_type_group(TRANSACTION_TYPE_GROUP.CORE)
+    transaction.set_nonce(1)
+    transaction.schnorr_sign('this is a top secret passphrase')
+    transaction.second_sign('second top secret passphrase')
+    transaction_dict = transaction.to_dict()
+
+    assert transaction_dict['nonce'] == 1
+    assert transaction_dict['signature']
+    assert transaction_dict['signSignature']
+    assert transaction_dict['type'] is TRANSACTION_TRANSFER
+    assert transaction_dict['typeGroup'] == 1
+    assert transaction_dict['typeGroup'] == TRANSACTION_TYPE_GROUP.CORE.value
+
+    transaction.schnorr_verify_second('second top secret passphrase') # if no exception is raised, it means the transaction is valid
```

Comment:
please add a `transaction.schnorr_verify` line above it to ensure it validates both signatures